### ISSUE TITLE
Crash Report Only After Joining Server

### ISSUE THREAD

**Issue by VirdsEyeBiew** [ldtteam/minecolonies#7252]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version: 0.14.222 ALPHA-universal

### Logs

https://gist.github.com/VirdsEyeBiew/b4f09feb370652e1932e2e1cc5594857

- latest.log: 

## Steps to reproduce the issue / details

Just reporting the AI issue. 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Update tinkers to latest
